### ISSUE TITLE
TypeError: Cannot read property 'voiceChannel' of undefined

### ISSUE THREAD

**Issue by AeroSouls** [galnir/Master-Bot#507]
**Describe the bug**
When you issue the command !play Playlistlinkyoutube it gives this error. Now, this only happens if you start with a playlist link(Before the bot is in VC). If I do !play nameofsong select number it works fine then I can do .play playlistlink works fine. only bugs if the first command it gets is a playlistlink. 

Once this first error hits the bot no longer works till restart. Ex. If I do the !play playlistlink first. now the !play songname no longer works because of the playlist bug. 

**Output from bot:** `TypeError: Cannot read property 'voiceChannel' of undefined`

**To Reproduce**
Start the bot, run the command !play https://www.youtube.com/playlist?list=PLgzTt0k8mXzEpH7-dOCHqRZOsakqXmzmG

or any playlist link.

This error occurs anytime you start the bot with the !play playlistlink command before its in VC

**Expected behavior**
It to join VC and play the playlist link.

**Screenshots**
Playlist error + after affect to the !play song command:

![image](https://user-images.githubusercontent.com/49828953/114929404-be4ff880-9e01-11eb-9bd1-ae4f92c59bd8.png)

**Desktop (please complete the following information):**
 - OS: Linux x64
 - Node Version(Should be v12 atleast): yes
 - Is python 2.7 installed?: Yes
 - How are you hosting the bot: Running on **Replit**

**Additional context**
I ran a fresh copy of the bot from GitHub had the same issue.

I searched the issues logs and have not found a soulution.

**Console log:**
```
(node:103) UnhandledPromiseRejectionWarning: DiscordAPIError: Unknown Message
    at RequestHandler.execute (/home/runner/TEsting/node_modules/discord.js/src/rest/RequestHandler.js:154:13)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async RequestHandler.push (/home/runner/TEsting/node_modules/discord.js/src/rest/RequestHandler.js:39:14)
    at async MessageManager.delete (/home/runner/TEsting/node_modules/discord.js/src/managers/MessageManager.js:126:5)
(node:103) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:103) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
Versions:
![image](https://user-images.githubusercontent.com/49828953/114930683-4da9db80-9e03-11eb-9f2a-77f35fcab836.png)

**Working after if you do !play songname first then adding the playlist after**
![image](https://user-images.githubusercontent.com/49828953/114931421-3ae3d680-9e04-11eb-8024-e4232580f402.png)

**Comment by galnir** [comment]
How exactly did you install python 2.7 if you're running the bot on replit...?

**Comment by AeroSouls** [comment]
> How exactly did you install python 2.7 if you're running the bot on replit...?

sorry not for Replit I have it installed on my host from pebblehost still get the same error

**Comment by AeroSouls** [comment]
My host has this package as well I have installed it and still having errors. 
https://github.com/73rhodes/node-python

**Comment by AeroSouls** [comment]
Tried it on another host plox.host still having the same error sadly

**Comment by Natemo6348** [comment]
also @galnir python is installed by default, and it seems that if you specify that the project is nodejs, it downgrades your node and python versions to 12.22.1 and 2.7.13

**Comment by galnir** [comment]
I'm able to reproduce this, looking into fixing it

**Comment by galnir** [comment]
Fixed in f582e1ef6535169deb36abc27f4111370cd1a3f2
Thank you @AeroSouls for opening this issue :)

**Comment by AeroSouls** [comment]
> Fixed in [f582e1e](https://github.com/galnir/Master-Bot/commit/f582e1ef6535169deb36abc27f4111370cd1a3f2)
> Thank you @AeroSouls for opening this issue :)

thanks, man I appreciate you and this bot!
